### PR TITLE
add mishap from 3 november 2020

### PR DESCRIPTION
**Pull request by thealphadollar**
This PR adds an account of the diagnosis that was done with the help of @iakshat on 3rd November in the mishap of no new notices being sent.

This will be useful in future if we face similar issues and have an account of problems (not necessarily bug fixes) that we have done in the past.

@iakshat Please let me know if I've missed any step that you'd like to be included.

@OrkoHunter You've been a pioneer in documentation in KOSS and I'd like to know your views on how I can make this initiative better.

### CODE REVIEW

File: error_diagnosis/3_Nov_2020.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+```
+shell> mongo $MONGODB_URI
+mongo> use CLIENT_STRING_FROM_END_OF_URI
+mongo> db.notices.find().sort({$natural: -1}).limit(11)
+```
+4. @iakshat suggested that it seems the member may not have permission as mailtrack was able to send the mails successfully. I checked for the presence of no-reply@mftp.herokuapp.com and it was not there. I confirmed the settings and only managers had the permission to post. So, @iakshat had pin pointed the error :fire:
+
+## The fix
+
+1. Confirm with @Ishita Lade and she was king enough to convey all the information. It was confirmed that all but emails given to her by @kaustubh were removed.
```
Comment (iakshat) [inline error_diagnosis/3_Nov_2020.md:19]:
```suggestion
1. Confirm with @Ishita Lade and she was king enough to convey all the information. It was confirmed that all but emails given to her by @kaustubh were removed.
```
1. Confirm with @Ishita Lade and she was kind enough to convey all the information. It was confirmed that all but emails given to her by @kaustubh were removed.

**Review by iakshat — APPROVED** [review]
Yeah it seems all the steps are covered really well covered!

**Review by OrkoHunter — APPROVED** [review]
Superb! You already wrote your first "post-mortem" as it's called. :)

**Review by icyflame — COMMENTED** [review]
Great initiative! ✅ 

1. I think we should call the folder `incident-reports` or `incident-postmortems`. `mishaps` or `error_diagnosis` is not a very common term
1. If we rename the file to have the string `yyyy-mm-dd`, then `ls -t` will automatically list the files in chronological or reverse chronological order, as we add more files to this folder
1. It might be a good idea to add a short summary as the filename stub: `incident-reports/2020-11-03-no-reply-email-removed-from-google-group.md`

File: error_diagnosis/3_Nov_2020.md (hunk 1 above)
Comment (thealphadollar) [reply error_diagnosis/3_Nov_2020.md:19]: Ooh! Thanks for noticing this tiny errors - I would have totally missed it :100:

**Comment by thealphadollar** [comment]
@icyflame @iakshat @OrkoHunter Thank you for your reviews - I've made some changes as follows:

- Changed folder name to "incident_postmortems".
- Added a README.md inside the folder with minimal guidelines and information.
- Added a template report for ease of writing in future.
- Update my report to follow the guidelines and formatting.

**Comment by icyflame** [comment]
@thealphadollar I meant specifically the `yyyy-mm-dd` format for the file name. It is good because it allows us to sort files and get the same order as they would be in real life. 

*For eg:*

```sh

$ cat <<EOF | sort
then> 2020-01-01
then> 2020-05-01
then> 2020-11-04
then> 2030-11-01
then> EOF
2020-01-01
2020-05-01
2020-11-04
2030-11-01

$ cat <<EOF | sort
then> 01-01-2020
then> 01-05-2020
then> 04-11-2020
then> 01-11-2020
then> 01-11-2030
then> EOF
01-01-2020
01-05-2020
01-11-2020
01-11-2030
04-11-2020
```

(This might seem like a nit, but I have found that naming files in `yyyy-mm-dd` is very useful when using `ls` or any Unix utility compared to any other format)

**Comment by thealphadollar** [comment]
@icyflame Thanks for explaining it - I didn't understand the reason fully in the beginning and thought we just wanted to have a better formatting. This scheme is actually pretty powerful in sorting and now that I realize that, it feels pretty good.

**Review by icyflame — APPROVED** [review]

**Pull request merged**